src/main/scala/io/sdkman/changelogs/JMeterMigration.scala

```diff
@@ -23,33 +23,20 @@ class JMeterMigration {
   }
 
   @ChangeSet(
-    order = "003",
-    id = "003-add_latest_jmeter_candidate_urls",
+    order = "005",
+    id = "005-add_5.5_jmeter_candidate",
     author = "xshyamx"
   )
-  def migration003(implicit db: MongoDatabase): List[Version] = {
-    List(
-      "5.4.3",
-      "5.4.2"
-    ).map(
-        version =>
-          Version(
-            candidate = "jmeter",
-            version = version,
-            url =
-              s"https://archive.apache.org/dist/jmeter/binaries/apache-jmeter-$version.zip"
-          )
-      )
-      .validate()
+  def migration003(implicit db: MongoDatabase): Unit = {
```

Comment:
Sorry for nitpicking but shouldn't this be `migration005`?

Reply:
Absolutely :). Copy paste error :(